lars-reimann/api-editor
# feat: Statistics View - Annotations

Closes #534.

### Summary of Changes
Added a /statisticsView page where the number of annotations for each annotation can be found. 

### Screenshots
![image](https://user-images.githubusercontent.com/32142988/174268923-304e37ee-c0bb-4d5f-af99-30a3b2c7dd98.png)


### Testing Instructions
Run api-edittor and klick on the Statistics View Button
check if the number of annotation for each type is correct
